api/src/main/java/com/cloud/network/router/VirtualRouter.java

```diff
@@ -35,7 +35,7 @@
     boolean getIsRedundantRouter();
 
     public enum RedundantState {
-        UNKNOWN, MASTER, BACKUP, FAULT
+        UNKNOWN, PRIMARY, BACKUP, FAULT
```

Comment:
@sureshanaparti as this value will be stored in the db, do we need to handle that or will it be taken care of as VRs may need to be recreated after the upgrade?